### ISSUE TITLE
Wrong docker image?

### ISSUE THREAD

**Issue by buckohfive** [antitree/private-tor-network#8]
As present now, the docker containers are all based on the antitree/private-tor-test.  These are not available to me (and likely no one except the author). Removing "-test" finds public images; however, this base image does not result in a successful private tor network. The DAs fail with a message of "TestingTorNetwork may only be configured in combination with a non-default set of DirAuthority or both of AlternateDirAuthority and AlternateBridgeAuthority configured." ... Do the -test base docker images correct this?

**Comment by mfrw** [comment]
I too faced the same issue, but what I did as a work around was, to check out an earlier branch (say maint-0.3.0). It worked.

**Comment by antitree** [comment]
@buckohfive @mfrw thanks. The typo was an easy fix but I'm wondering if the other problem was because I've migrated from `docker-compose` to `docker swarm` but didn't update the documentation. I re-added the old docker-compose.yml that supports `docker-compose` as "docker-compose.2.yml" which should work as previously advertised. 

I've also updated the details about using swarm in the README. The biggest changes is that I'm using a named volume instead of mapping a local folder so you won't see anything show up in your `./tor` folder. 

I just tested 0.3.1 and it seems to be working but feel free to open another ticket if you still see a problem. That might take more time to track down.
